x-pack/plugins/cases/public/components/markdown_editor/eui_form.tsx

```diff
@@ -54,6 +59,7 @@ export const MarkdownEditorForm: React.FC<MarkdownEditorFormProps> = ({
           value={field.value as string}
           data-test-subj={`${dataTestSubj}-markdown-editor`}
         />
+        {errorMessage && isInvalid && <EuiFormErrorText>{errorMessage}</EuiFormErrorText>}
```

Comment:
I think is best to move the `bottomRightContent` outside the `EuiFormRow` and keep the `error` prop. I think is better to leave the responsibility to `EuiFormRow` in case of future updates to the UI library.

```
<> // You can change it with EuiFlexGroup if needed.
      <EuiFormRow
        data-test-subj={dataTestSubj}
        describedByIds={idAria ? [idAria] : undefined}
        error={errorMessage}
        fullWidth
        helpText={field.helpText}
        isInvalid={isInvalid}
        label={field.label}
        labelAppend={field.labelAppend}
      >
        <>
          <MarkdownEditor
            ariaLabel={idAria}
            editorId={id}
            onChange={field.setValue}
            value={field.value as string}
            data-test-subj={`${dataTestSubj}-markdown-editor`}
          />
        </>
      </EuiFormRow>
      {bottomRightContent && (
        <BottomContentWrapper justifyContent={'flexEnd'}>
          <EuiFlexItem grow={false}>{bottomRightContent}</EuiFlexItem>
        </BottomContentWrapper>
      )}
    </>
```

Reply:
I agree. That is a better solution.